tests/functional/adapter/materialized_view_tests/test_materialized_views.py

```diff
@@ -108,10 +108,26 @@ def change_config_via_alter(project, materialized_view):
         new_model = initial_model.replace("dist='id',", "dist='id', auto_refresh=True")
         set_model_file(project, materialized_view, new_model)
 
+    @staticmethod
+    def change_config_via_alter_str_true(project, materialized_view):
```

Comment:
We might be able to make some slight improvements in this method and the `check_state_alter_is_applied` method. Possibly by just making the original method take in the third keyword arg, we can change in the tests. I tried doing it with f-strings but it didn't seem to work. any thoughts on this @mikealfare?

Reply:
I think what may be complicating this is trying to test the various inputs via the change monitoring tests. Have you considered a more focused test? We effectively want to test different model config inputs; so a class that creates a bunch of new models with different settings for `auto_refresh` would suffice.